srcpkgs/singularity/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'singularity'
+pkgname=singularity
+version=3.2.1
+revision=1
+archs="~arm* ~aarch*"
+build_style=configure
+hostmakedepends="go git pkg-config"
+makedepends="gpgme-devel libressl-devel libuuid-devel libseccomp-devel"
+depends="squashfs-tools"
+short_desc="Open source container platform designed to be simple, fast, and secure"
+maintainer="Emilio Palumbo <emiliopalumbo@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://www.sylabs.io/singularity"
+distfiles="https://github.com/sylabs/singularity/releases/download/v${version}/singularity-${version}.tar.gz"
+checksum=d4388fb5f7e0083f0c344354c9ad3b5b823e2f3f27980e56efa7785140c9b616
+nopie=yes
+
+wrksrc="singularity"
+configure_script="./mconfig"
+configure_args="--sysconfdir=/etc --localstatedir=/var --runstatedir=/run --prefix=/usr"
+make_build_args="-C builddir"
+make_install_args+="-C builddir"
+make_install_target="install man"
+make_dirs="/var/singularity/mnt/session 0750 root root"
+
+pre_install() {
+	vmkdir usr/share/man/man1
+}
+
+post_install() {
+	vlicense LICENSE.md
+	vdoc README.md
+	vdoc CHANGELOG.md
+	vcopy examples usr/share/doc/${pkgname}/
+	vmkdir usr/share/bash-completion/completions/
+	mv $DESTDIR/etc/bash_completion.d/singularity $DESTDIR/usr/share/bash-completion/completions/
+	rmdir $DESTDIR/etc/bash_completion.d/
```

Comment:
Is it intended to fail when this directory contain more files? if not, use `rm -r`